pay-api/src/pay_api/services/flags.py

```diff
@@ -59,12 +60,6 @@ def init_app(self, app):
             client = ldclient_get()
 
             app.extensions['featureflags'] = client
-            app.teardown_appcontext(self.teardown)
-
-    def teardown(self, exception):  # pylint: disable=unused-argument; flask method signature
```

Comment:
This teardown needed to be removed, otherwise flags get stuck being cached.  Everytime it evaluates it hits this warning:

![image](https://user-images.githubusercontent.com/3484109/207739087-e3d31aba-e083-4816-b6a7-69ded3936057.png)